### DISCUSSION TITLE
序盤用に★武器を報酬にしたクエストを作りたい

### DISCUSSION THREAD

**Discussion by dis-** [hengband/hengband#348] [Ideas]
イメージは簡単にした宝物庫クエスト。
オークキャンプの前後で受注できる難易度を想定している。

報酬として決められた★武器リストからランダムに一つ与えられる。
出番のない★武器に活躍の場を与えていきたい。

報酬にするべき★、クエストの詳細、受注する場所と受注する順番など、詳細を詰めていきたい。

**Comment by dis-** [comment]
案として出た候補を並べていく。

★丈高きマエズロスのマン・ゴーシュ (2d5) (+12,+15)
    +3の修正: 知能, 器用さ, スピード
    対: トロル, 巨人
    麻痺知らず, 可視透明, オーク感知, トロル感知, 巨人感知
    追加: 能力or耐性
    発動: なし
    レベル 15, 希少度 30, 1.5 kg, ＄62500

★リリアのダガー (2d4) (+4,+3)
    対: オーク
    武器属性: 毒殺
    耐性: 毒, 劣化
    投擲
    発動: 悪臭雲(12) : 4 ターン毎
    レベル 5, 希少度 40, 0.6 kg, ＄35000

★ダガー『グリムトゥース』 (1d4) (+5,+5)
    +1の修正: 器用さ, 攻撃回数
    武器属性: 毒殺
    投擲
    発動: なし
    レベル 7, 希少度 15, 0.6 kg, ＄25000

★マン・ゴーシュ『小さき刺』 (1d5) (+18,+6)
    +3の修正: 知能, 魅力
    対: 邪悪
    耐性: 因果混乱
    麻痺知らず, 可視透明, 急速回復
    発動: なし
    レベル 20, 希少度 6, 1.5 kg, ＄22500

★農夫マゴットの鎌 (2d3) (+7,+9)
    +1の修正: 腕力, 器用さ
    武器属性: 切れ味
    耐性: 火炎, 閃光
    発動: なし
    レベル 10, 希少度 4, 3.5 kg, ＄10000

★怪力サムソンの棍棒 (2d4) (+8,+10)
    +3の修正: 腕力, 耐久力
    対: 邪悪, アンデッド
    耐性: 恐怖
    可視透明
    発動: なし
    レベル 10, 希少度 20, 2.5 kg, ＄40000

★ジョーンズ博士のムチ (1d6) (+16,+13)
    +1の修正: 知能, 賢さ
    浮遊, 可視透明
    発動: 物体を引き寄せる(重量25kgまで) : 25+d25 ターン毎
    レベル 8, 希少度 5, 1.5 kg, ＄18000

**Comment by RL-Nadir** [comment]
ワーグクエストの応用みたいな感じで、以下を提案します。

・辺境の地にて、ワーグ、物言えぬ証人以降のクエストとする。
・イーク洞のボスは『オルファックス』とする
・クエスト受注で最下層に『ボルドール』が出現するようになり、これの討伐を達成条件とする（倒すまで失敗にならない）
・報酬はこの話題で議論された微妙★郡からランダム
・召喚持ち、護衛持ちの捌き方の勉強とそのご褒美としては悪くなく、且つ大して強くはない★が並びそうだし、中級者以上なら適宜無視するだろうという点がこの提案の理由

2/28追記
・報酬受領時に広域マップと鉄獄、他の街について言及。あなたの冒険はこれからだ、頑張ってね、的な。
・合わせて、鉄獄と他の街のマップ登録も？
・まあ、このゲームが初心者に寄り添う意味ってどれだけあるのかという気もしますが。。。

**Comment by RRyukkuriRyu** [comment]
discordで言っていた、激安の殿堂クエスト1の話題を参考に書いてみました。あくまでも案なので改変していただいて構いません。
・オークキャンプの前後想定なら鉄獄６RQは攻略してそうなのでモリバントで受注。
・クエスト受注場所は、戦士の集会所がクエスト枠だけ存在して受注できるクエストがない状態なのでそこに入れれば既存のクエスト前にわざわざ攻略or破棄する必要がなくて丁度いい。
・MAPはドンキの圧縮陳列を再現した狭い通路で構成された中規模な部屋が2つ、報酬の★が落ちている小規模な部屋が1つ、discordで言っていた誰得の巻物置き場が1つ、それらを繋ぐ通路で構成　あくまでも★の入手がクリア条件なのでルートは複数用意　低隠密ビルドは通路で1対1できる
・前半は雑魚の群れ、中盤は召喚や元素攻撃で少し厄介な雑魚、★部屋は若干階層が深めで強いがパニモンが効く雑魚で守らせる配置
・入り口付近以外テレポ禁止エリア　逆に言えば★回収後テレポで即入り口に戻れる　クエスト受注文にもテレポで戻れることを記載してもいいかも
イメージ図↓
　　　　　　　　　　　　　　　　　　　　　　　　　＃＃＃
　　　　　　　　　　　　　　　　　　　　　　　　　＃★＃
　　　　　　　　　　　　　　　　　　　　　　　　＃＃Ｈ＃＃
　　　　　　　　　　　　　　　　　　　　　　　　＃＾＾＾＃
　　　　　　　　　　　　　　　　　　　　　　＃＃＃＋＋＋＃＃＃
　　　　　　　　　　　　　　　　　　　　　　＃＄＄＄＄＄＄＄＃
　　　　　　　　　　　　　　　　　　　　　　＃＋＃＃＋＃＃＋＃
　　　　　　　　　　　　　　　　　　　　　　＃　　　　　　　＃
　　　　　　　　　　　　　　　　　　　　　　＃ｋ＃ｋ＃ｋ＃ｋ＃
　　　　　　　　　　　　　　　　　　　　　　＃　＃　＃　＃　＃
　　　　　　＃＃＃＃＃＃＃＃＃＃＃＃＃＃＃＃＃ｋ＃ｋ＃ｋ＃ｋ＃
＃＃＃＃　　＃　ｏ　ｏ　ｏ　ｏ　ｏ　ｏ　ｏ　＃　＃　＃　＃　＃
＃　　＃＃＃＃　＃＃＃＃＃＃＃＃＃＃＃＃＃　＃ｋ＃ｋ＃ｋ＃ｋ＃
＃＠　＋　　＋　ｏ　ｏ　ｏ　ｏ　ｏ　ｏ　ｏ　＃　＃　＃　＃　＃
＃　　＃＃＃＃　＃＃＃＃＃＃＃＃＃＃＃＃＃　＃ｋ＃ｋ＃ｋ＃ｋ＃
＃＃＃＃　　＃　ｏ　ｏ　ｏ　ｏ　ｏ　ｏ　ｏ　＃　　　　　　　＃
　　　　　　＃＃＃＃＃＃＃＃＃＃＃＃＃＃＃＋＃＃＃＃＋＃＃＃＃
　　　　　　　　　　　　　　　　　　　　＃ｓｓｓｓｓｓ＃
　　　　　　　　　　　　　　　　　　　　＃＃＃＃＃＃＋＃＃＃＃
　　　　　　　　　　　　　　　　　　　　　　　　　＃？？？？＃
　　　　　　　　　　　　　　　　　　　　　　　　　＃？？？？＃
　　　　　　　　　　　　　　　　　　　　　　　　　＃？？？？＃
　　　　　　　　　　　　　　　　　　　　　　　　　＃＃＃＃＃＃
　　　　　　　　　　　　　　　　　　　　　　　　　
＃：永久壁　何列か陳列棚っぽく永久ガラスにすると見栄えがいいかも
＋：ドア
＾：警報装置
?：誰得の巻物
o：スナガなど雑魚の群れ 列によって種類を変えたりイークなど別シンボルを配置するのもありかも
s：窖に潜むもの　召喚持ち
k：下位元素コボルド　左から火冷電酸　毒は若干強いので除外
$：クリーピングコイン　銅or銀or金　お好みで
Ｈ：白ワニや『迷宮のミノタウロス』的な、殴りが強いけどパニモンが効くレベルの敵　マンティコアとか
★：クエスト目標
ランダムモンスターなし
